fortytools.yaml

```diff
@@ -3207,14 +3206,12 @@ components:
         `sha2($client_user_api_token + ':' + $app_token + ':' + $validuntil)`
       name: signature
       in: query
-    ClientCredentials:
-      type: oauth2
-      description: look at Oauth documentation
-      flows:
-        clientCredentials:
-          tokenUrl: https://app.fortytools.com/api/v2/token/
-          scopes: {}
-    BearerAuth:
+    bearerAuth:
       type: http
-      scheme: Bearer
-      description: get access token from oauth2
+      scheme: bearer
+      description: |
```

Comment:
I would suggest to add a general description somewhere (maybe [here](https://github.com/fortytools/fortytools.github.io/blob/master/fortytools.yaml#L52)) to how the whole authentication flow looks like

1. getting uid and secret from client settings in app.fortytools.com
2. getting access token from ...
3. use this access token in each API request as a header `Authorization: Bearer .....`

And by the way in case the toke expires, how would the client get a new token? should they call the same endpoint with `client_id` and `client_secret` again? Please document this as well